docs/rules.md

```diff
@@ -558,15 +558,6 @@ Feature: Foobar
     @tag4 @tag5
     Scenario: Barfoo
 ```
-**Good**: Tags have one space between them
```

Comment:
What made this change? I didn't do that (manually). :thinking:

Reply:
this is generated: `"GHERKINLINT_DEV=1 bin/gherkinlint ruledocs docs/rules.md"` (see `composer.json`)

Reply:
so maybe it's a regression somehow?!

Reply:
Let me check.

Reply:
Yes, it happens on master too for me.

Reply:
I've removed the change to the docs file, Psalm is still fixed, the regression is not related to this PR.